### ISSUE TITLE
Create BATS tests for BATS helper functions

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#5130]
Our BATS helper functions have become quite complex. We need to create some kind of unit tests for them, to make sure refactoring the helpers doesn't have unintended effects on our existing tests.

**Comment by jandubois** [comment]
#5134 implements unit tests for the current helper functions, but once #4991 is merged we need to add tests for all the profile functions and any other new helper functions added there.

**Comment by jandubois** [comment]
> once #4991 is merged we need to add tests for all the profile functions and any other new helper functions added there.

I've added a separate task for this to the epic, so this issue can be closed once #5134 is merged.

**Comment by gaktive** [comment]
Closing this because work is done.
